sys-auth/libfprint/files/libfprint-1.94.9-tod-fix-linking-with-lld.patch

```diff
@@ -0,0 +1,23 @@
+https://gitlab.freedesktop.org/3v1n0/libfprint/-/issues/10
+From: Brahmajit Das <listout@listout.xyz>
+Date: Sat, 20 Sep 2025 10:14:33 +0530
+Subject: [PATCH] fix fpi_byte_writer_change_pos not defined with clang
+
+On clang, linking with LLD fails.
+ld.lld: error: version script assignment of 'LIBFPRINT_TOD_1_1.94' to symbol 'fpi_byte_writer_change_pos' failed: symbol not defined
+clang: error: linker command failed with exit code 1 (use -v to see invocation)
+
+Signed-off-by: Brahmajit Das <listout@listout.xyz>
+--- a/libfprint/tod/libfprint-tod.ver.in
++++ b/libfprint/tod/libfprint-tod.ver.in
+@@ -238,7 +238,6 @@ global:
+ 
+ LIBFPRINT_TOD_@tod_soversion@_1.94 {
+ global:
+-    fpi_byte_writer_change_pos;
```

Comment:
Changes like this should really have more analysis. The fact that lld fails on this does not mean you should just remove it.

The questions are:
1) Did `fpi_byte_writer_change_pos` ever exist as a symbol?
2) Is it defined only conditionally?
3) Is it supposed to be exported upstream?